src/util/getRandomUrl.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Helper method for generating a randomized valid url string.
+ *
+ * This is useful for testing links that have a distinct visited style. If the link always has
+ * the same href, you can only test the non-visited state once (unless you're using incognito
+ * browser windows) because once you've visited the site, the link now uses the visited style.
+ * And it needs to be a valid url in order to visit the site and trigger the visited state.
+ */
+const getRandomUrl = () => {
```

Comment:
I like this lol. Think faker had a url generator but would rather not import it and keep it lighter like this. Also faker had that [corrupt package release](https://www.theverge.com/2022/1/9/22874949/developer-corrupts-open-source-libraries-projects-affected) so not keen on using it